### DISCUSSION TITLE
Why is "diff" rendering endpoint better for SVGs than "added" rendering endpoint?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22692] [New to GitHub]
<p>If you compare <a href="https://github.com/PKief/vscode-material-icon-theme/pull/707/files?short_path=9ceb7fe#diff-9ceb7fe274a1718f15b57850e7790036" rel="nofollow noopener">this diff using the added endpoint</a> and <a href="https://github.com/PKief/vscode-material-icon-theme/pull/707/commits/ba0b7ab5ba2f70815677ad469e01f3d1d898d5bb?short_path=9ceb7fe#diff-9ceb7fe274a1718f15b57850e7790036" rel="nofollow noopener">this diff using the diff endpoint</a>, the second displays much better. The second link shows a buggy and unviewable diff… So why has Github not consolidated the endpoints?</p>

**Comment by yamiacat** [comment]
<p>Hi <a class="mention" href="/lmn8">@lmn8</a>! :wave:</p>
<p>I’m not sure I understand what you mean -   you say “The second link shows a buggy and unviewable diff” but presumably you mean the first link?</p>
<p>I think this is due to what’s being shown? The first link is looking at all commits, so is comparing the initial state with no icons to the final state of the PR, with the chosen icons. Since there’s only one version of an image, you don’t see the options to do with diffing images:</p>
<p><a href="https://help.github.com/en/github/managing-files-in-a-repository/rendering-and-diffing-images" class="onebox" target="_blank">https://help.github.com/en/github/managing-files-in-a-repository/rendering-and-diffing-images</a></p>
<p>In the second view you’re diffing a subset of the commits that have two different versions of the images, so that’s why you see the fancier image diffing tools.</p>
<p>Does that answer your question?</p>

**Comment by ghost** [comment]
<p>Yes! I think new SVG icons without a “height” and “width” should really be rendered 150px by 150px for consistency &amp; convenience.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Oh, I see! We’re always working to improve GitHub, and we consider every suggestion we receive, so perhaps you’d like to submit a feature request <a href="https://support.github.com/contact/feedback">through our official product feedback form</a> so that our product team can see exactly how you’d like image diffing on GitHub to work?</p>

**Comment by ghost** [comment]
<p>Thanks for adding the feature. This thread probably can be closed now</p>
